Tools/CISupport/ews-build/steps.py

```diff
@@ -5929,6 +5930,83 @@ def getResultSummary(self):
             configuration += ', Xcode: {}'.format(xcode_version)
         return {'step': configuration}
 
+class CheckOutLLVMProject(git.Git, AddToLogMixin):
```

Comment:
Check if we need to run this step of checking out this repo even when we are not going to update clang? If not, you can consider adding doStepIf similar to what you have in UpdateClang.doStepIf(), and run this step after `print-clang-version` step.

Reply:
Very good point!